sei/on_poll.py

```diff
@@ -1,21 +1,22 @@
 import phantom.app as phantom
-
-from .get_events import get_events
+from phantom.action_result import ActionResult
+import sei
 
 def on_poll(connector, param):
-    timestamp = connector._state["last_poll"] if "last_poll" in connector._state.keys() else '2022-08-01T00:00:01Z'
+    timestamp = connector._timestamp if connector._timestamp != None else connector._state["last_poll"] if "last_poll" in connector._state.keys()  else '2022-08-01T00:00:01Z'
     container_id = connector._state["container_id"] if "container_id" in connector._state.keys() else None
-    
+    print(timestamp)
     if container_id is None or connector.get_container_info(container_id)[0] == False:
         container = {'name': 'WithSecure - security events', 'label': 'events',
-                     'container_type': 'default'}
+                     'container_type': 'default', "run_automation": True}
         success, message, container_id = connector.save_container(container, fail_on_duplicate=True)
 
-
-    artifacts, new_timestamp, action_result = get_events(connector, param, container_id, timestamp)
+    action_result = connector.add_action_result(ActionResult(dict(param)))
```

Comment:
`action_result` should be passed by params. It is passed in the connector please add it to function parameters, adjust the test and check if it works in Splunk.